### ISSUE TITLE
Error: Trying to copy from a source that does not exist

### ISSUE THREAD

**Issue by ericglau** [onlydustxyz/generator-starknet#21]
Installed the package using npm.  With any version of the package, after going through the options, the following error occurs:

```
Error starknet 

Trying to copy from a source that does not exist: /Users/<username>/.nvm/versions/node/v16.13.1/lib/node_modules/generator-starknet/generators/app/templates/.gitignore
```

This looks like a packaging issue for [the .gitignore file in the template](https://github.com/onlydustxyz/generator-starknet/blob/main/generators/app/templates/.gitignore).  The file appears as `.npmignore` rather than `.gitignore` in the node modules.

Possible solutions may be to [do something with .npmignore](https://stackoverflow.com/questions/24942161/does-npm-ignore-files-listed-in-gitignore) or just change the template file name to something else, then change it to `.gitignore` while generating the actual project.

**Comment by abdelhamidbakhta** [comment]
> 

Nice catch Eric. Would be cool if you could submit a PR to fix, either solution are good.

**Comment by ericglau** [comment]
@abdelhamidbakhta Any plans to release a new version? Since otherwise this package doesn't work when installed through npm, it only works with yarn at the moment. Thanks!

**Comment by abdelhamidbakhta** [comment]
> @abdelhamidbakhta Any plans to release a new version in the near term? Since otherwise this package doesn't work when installed through npm, it only works with yarn at the moment. Thanks!

I released 0.1.5 version. Let me know if you have any isssues with it
